File: c#/uri/1006.cs
using System;
class URI{
    static void Main(){
        float a = float.Parse(Console.ReadLine().Trim());
        float b = float.Parse(Console.ReadLine().Trim());
        float c = float.Parse(Console.ReadLine().Trim());
        double d = ((a * 2) + (b * 3) + (c * 5))/10;
        Console.WriteLine("MEDIA = {0:f1}", d);
    }
}

File: js/URI/1006.js
var input = require('fs').readFileSync('URI/in', 'utf-8');
var lines = input.split('\n');
var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());
var d = ((a * 2) + (b * 3) + (c * 5))/10;
var f = d.toFixed(1);
console.log(`MEDIA = ${f}`);
